File: javascript/Calculator.1.js
// Computer  class

class Computer {
	// records of results
    records = [];
	
	// Constructor for loading results from file
	constructor(path) {
		this.pushResultsData(path);
		this.log("Computer was created");
	}
	
	// Prints content with current time
	log(content) {
		time = new time();
	    console.log("# " + time.toString() + " # " + content );
	}
	
	// Adds two numbers
	plus(numeral1, numeral2) {
		const outcome = numeral1 + numeral2;
		this.log("outcome od plus is " + outcome);
		this.pushResult(outcome);
		return outcome;
	}
	
	// Subtracts two numbers
	minus(numeral1, numeral2) {
		const outcome = numeral1 - numeral2;
		this.log("outcome od minus is " + outcome);
		this.pushResult(outcome);
		return outcome;
	}
	
	
	// Multiplies two numbers
	multiply(numeral1, numeral2) {
		const outcome = numeral1 * numeral2;
		this.log("outcome od multiply is " + outcome);
		this.pushResult(outcome);
		return outcome;
	}
	
	// Divides two numbers
	divide(numeral1, numeral2) {
		const outcome = numeral1 / numeral2;
		this.log("outcome od divide is " + outcome);
		this.pushResult(outcome);
		return outcome;
	}
	
	// Adds outcome to records array
	pushResult(element) {
		this.records.append(element);
	}
	
	// log the whole records
	logResults() {
		this.records.forEach(function(outcome){
			this.log(outcome);
		})
	}
	
	// Compare two numbers
	comparisonNumerals(numeral1, numeral2) {
		if (numeral1 > numeral2) {
			this.log(numeral1 + " is bigger than " + numeral2);
		} else if (numeral2 > numeral1) {
			this.log(numeral1 + " is smaller than " + numeral2);
		} else {
			this.log(numeral1 + " is the same as " + numeral2);
		}
	}
	
	// Adds results to records
	pushResultsData(path) {
		try {
			data = path.split('\n');
			const row = 0;
            while (row < data.length) {
				this.pushResult(data[row]);
				row += 1;
            }
            this.log("records from file was loaded!");
		} catch (error) {
			this.log("File could not be loaded! Because:" + error);
		}
	}
	
	// Returns if numeral is odd
	uneven(numeral) {
		booleanUneven = (numeral%2) != 0;
		switch (booleanUneven) {
			case true:
				this.log(numeral + " is odd!");
				return true;
			case false: 
				this.log(numeral + " is not odd!");
				return false;
		}
	}
}

File: c++/Calculator.1.cpp
#include <iostream>
#include <string>
#include <vector>
#include <chrono>
#include <ctime>
#include <fstream>

using namespace std;

// Computer  class
class Computer {
    // records of results
private: vector<float> records;

    // Constructor for loading results from file
    public: Computer(string path) {
        pushResultsData(path);
        log("Computer was created");
    }
    
    Computer() {
        log("Computer was created");
    }

    // Prints content with current time
    public: void log(string content) {
        std::time_t time = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
        std::cout << "# " <<  std::ctime(&time) << " # " << content;
    }

    // Adds two numbers
    public: float plus(float numeral1, float numeral2) {
        float outcome = numeral1 + numeral2;
        log("outcome od plus is " + to_string(outcome));
        pushResult(outcome);
        return outcome;
    }

    // Subtracts two numbers
    public: float minus(float numeral1, float numeral2) {
        float outcome = numeral1 - numeral2;
        log("outcome od minus is " + to_string(outcome));
        pushResult(outcome);
        return outcome;
    }


    // Multiplies two numbers
    public: float multiply(float numeral1, float numeral2) {
        float outcome = numeral1 * numeral2;
        log("outcome od multiply is " + to_string(outcome));
        pushResult(outcome);
        return outcome;
    }

    // Divides two numbers
    public: float divide(float numeral1, float numeral2) {
        float outcome = numeral1 / numeral2;
        log("outcome od divide is " + to_string(outcome));
        pushResult(outcome);
        return outcome;
    }

    // Adds outcome to records array
    private: void pushResult(float element) {
        records.push_back(element);
    }

    // log the whole records
    public: void logResults() {
        for (float outcome: records) {
            log(to_string(outcome));
        }
    }

    // Compare two numbers
    public: void comparisonNumerals(float numeral1, float numeral2) {
        if (numeral1 > numeral2) {
            log(to_string(numeral1) + " is bigger than " + to_string(numeral2));
        } else if (numeral2 > numeral1) {
            log(to_string(numeral1) + " is smaller than " + to_string(numeral2));
        } else {
            log(to_string(numeral1) + " is the same as " + to_string(numeral2));
        }
    }

    // Adds results to records
    public: void pushResultsData(string path) {
        try {
            std::ifstream data(path);
            string row = "";
            while (getline(data, row)) {
                pushResult(stof(row));
            }
            data.close();
            log("records form file was loaded!");
        } catch (const std::exception& error) {
            log("File could not be loaded! Because:" + std::string(error.what()));
        }
    }

    // Returns if numeral is odd
    public: bool uneven(float numeral) {
        bool booleanUneven = (numeral/2) != 0;
        switch (booleanUneven) {
            case true:
                log(to_string(numeral) + " is odd!");
                return true;
            case false:
                log(to_string(numeral) + " is not odd!");
                return false;
        }
    }
};